fedgov-cv-ISDcreator-webapp/src/main/webapp/css/switch.css

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
Can you please mention that this CSS file is from the previous PR? This way it is easier for @saina-ramyar to review the PR.

Reply:
@DokurOmkar  Added